cocos/2d/renderer/batcher-2d.ts

```diff
@@ -273,92 +231,58 @@ export class Batcher2D implements IBatcher {
     }
 
     public uploadBuffers (): void {
-        if (JSB) {
-            this._nativeObj.uploadBuffers();
-        } else if (this._batches.length > 0) {
+        if (this._batches.length > 0) {
             const length = this._meshDataArray.length;
             for (let i = 0; i < length; i++) {
                 this._meshDataArray[i].uploadBuffers();
             }
 
-            for (const accessor of this._bufferAccessors.values()) {
-                accessor.uploadBuffers();
-                accessor.reset();
-            }
+            BufferAccessorManager.instance.upload();
```

Comment:
Seems better using UISystem to control BufferAccessorManager. No need to couple batcher and BufferAccessor.